scripts/lib/CIME/provenance.py

```diff
@@ -250,32 +275,23 @@ def save_postrun_provenance_acme(case, lid):
             finally:
                 os.remove(syslog_jobid_path)
 
-    # copy/tar timings
-    rundir_timing_dir = os.path.join(rundir, "timing." + lid)
-    shutil.move(os.path.join(rundir, "timing"), rundir_timing_dir)
-    with tarfile.open("{}.tar.gz".format(rundir_timing_dir), "w:gz") as tfd:
-        tfd.add(rundir_timing_dir, arcname=os.path.basename(rundir_timing_dir))
-
-    shutil.rmtree(rundir_timing_dir)
-    copy_umask("{}.tar.gz".format(rundir_timing_dir), full_timing_dir)
-
-    gzip_existing_file(os.path.join(caseroot, "timing", "acme_timing_stats.{}".format(lid)))
-
-    # JGF: not sure why we do this
-    timing_saved_file = "timing.{}.saved".format(lid)
-    touch(os.path.join(caseroot, "timing", timing_saved_file))
+    # copy timings
+    copy_umask("%s.tar.gz" % rundir_timing_dir, full_timing_dir)
 
     #
     # save output files and logs
     #
     globs_to_copy = []
-    if mach == "titan":
-        globs_to_copy.append("{}*OU".format(job_id))
-    elif mach == "mira":
-        globs_to_copy.append("{}*output".format(job_id))
-        globs_to_copy.append("{}*cobaltlog".format(job_id))
-    elif mach in ["edison", "cori-haswell", "cori-knl"]:
-        globs_to_copy.append("{}".format(case.get_value("CASE")))
+    if job_id is not None:
```

Comment:
Same comment as above. On the bright side, CESM doesn't need to worry about this code.